js/bootstrap-slider.js

```diff
@@ -888,11 +897,21 @@
 				this.sliderElem.removeEventListener("mousedown", this.mousedown, false);
 			},
 			_bindNonQueryEventHandler: function(evt, callback) {
-				if(this.eventToCallbackMap[evt]===undefined) {
+				if(this.eventToCallbackMap[evt] === undefined) {
 					this.eventToCallbackMap[evt] = [];
 				}
 				this.eventToCallbackMap[evt].push(callback);
 			},
+            _unbindNonQueryEventHandler: function(evt, callback) {
+                if(this.eventToCallbackMap[evt] !== undefined) {
```

Comment:
at the top of this function, I would put `this.eventToCallbackMap[evt]` into a variable and then reference that throughout the function. 

```js
var boundCallbacks = this.eventToCallbackMap[evt];
```